lib/src/screens/dashboard/dashboard_page.dart

```diff
@@ -125,7 +125,9 @@ class DashboardPageBodyState extends State<DashboardPageBody> {
     final actionListStore = Provider.of<ActionListStore>(context);
     final syncStore = Provider.of<SyncStore>(context);
     final settingsStore = Provider.of<SettingsStore>(context);
-    final transactionDateFormat = DateFormat("MMMM d, yyyy HH:mm", settingsStore.languageCode);
+    final transactionDateFormat = settingsStore.currentLocale == 'en_US'
+          ? DateFormat("MMMM d, yyyy, HH:mm", settingsStore.languageCode)
```

Comment:
Move logic for selection of dateformat to another place. For example you can create factory which will accept current locale (or settings store if it contains current local) and will return DateFormat object. Or you can make same as a function which will do same. Do same for all another places where you are doing same (final df = isUSALocal ? DateFormat(...) : DateFormat(...)).